src/codegeneration/ModuleTransformer.js

```diff
@@ -276,8 +276,8 @@ export class ModuleTransformer extends TempVarTransformer {
     assert(this.moduleName);
     let name = tree.token.processedValue;
     // import/module {x} from './name.js' is relative to the current file.
-    let normalizedName = System.normalize(name, this.moduleName);
-    return parseExpression `System.get(${normalizedName})`;
+    let normalizedName = $traceurRuntime.normalizeModuleName(name, this.moduleName);
```

Comment:
YES WE THINK OF THIS AS THE DIFFERENCE BETWEEN A BUILD AND A BUNDLE IN SYSTEMJS. A BUILD CAN USE A LOADER INTERNALLY, BUT IT JUST DOESN'T EXPOSE THE LOADER GLOBALLY.